sse2neon.h

```diff
@@ -896,6 +899,20 @@ FORCE_INLINE __m128 _mm_andnot_ps(__m128 a, __m128 b)
                   vreinterpretq_s32_m128(a)));  // *NOTE* argument swap
 }
 
+// Compute the bitwise NOT of packed double-precision (64-bit) floating-point
+// elements in a and then AND with b, and store the results in dst.
+//
+//   r0 := ~a0 & b0
+//   r1 := ~a1 & b1
```

Comment:
Shall we replace it with the **Operation** on [_mm_andnot_pd](https://software.intel.com/sites/landingpage/IntrinsicsGuide/#text=_mm_andnot_pd)?

@jserv what do you think?

Reply:
I agree. Let's copy the description on operations from Intel website.

Reply:
Quote from Intel:
```
FOR j := 0 to 1
	i := j*64
	dst[i+63:i] := ((NOT a[i+63:i]) AND b[i+63:i])
ENDFOR
```